Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1221,6 +1228,10 @@ internal extension Purchases {
         return self.userDefaults
     }
 
+    var publicKey: Signing.PublicKey? {
+        return self.systemInfo.responseVerificationLevel.publicKey
+    }
```

Comment:
I guess this will be used in future tests?

Reply:
Oh I moved these tests somewhere else, but let me add another one for `Purchases.Configuration` that uses this too.
I moved them out on a refactor and also when I made this `internal`, but they're still useful for when we make it `public`.